dts/bindings/ethernet/snps,ethernet-cyclonev.yaml

```diff
@@ -0,0 +1,17 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (C) 2022, Intel Corporation
+# Description:
+# Binding file for Ethernet driver CycloneV
+# heavily modified for Zephyr
+
+description: Synopsys DesignWare Ethernet
+
+compatible: "snps,ethernet-cyclonev"
```

Comment:
This compatible doesn't feel right.  Is there something cyclonev specific about the ethernet device?  Is this not some `snps,dwmac-<IP_VER>`?

https://git.kernel.org/pub/scm/linux/kernel/git/devicetree/devicetree-rebasing.git/tree/Bindings/net/snps,dwmac.yaml

Reply:
Actually the one we use for linux is this one:
https://git.kernel.org/pub/scm/linux/kernel/git/devicetree/devicetree-rebasing.git/tree/Bindings/net/socfpga-dwmac.txt

I can change the naming, but this is up to you...

Reply:
Which is compatible with CycloneV, Arria10, Stratix10... But I dont want to use "socfpga" as some other FPGAs might not work with it (Like Agilex)

Reply:
@galak just to be 100% clear, should I go ahead an change the naming to "altr,socfpga-stmmac" everywhere?